pkg/devices/holding_register.go

```diff
@@ -45,6 +47,19 @@ var HoldingRegisterHandler = sdk.DeviceHandler{
 	},
 }
 
+// ReadOnlyHoldingRegisterHandler is a device handler used to read modbus
+// holding registers. No write method is available.
```

Comment:
Could add a line saying that this can be used when wanting to restrict write access to holding registers just to make it clear that writes are disabled intentionally, not because the holding registers don't support writing. I feel like this is already implied, but  may be worth calling out more explicitly.